action/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3
```

Can we check if this works with the slim image? Found it last week. Big wins.
```suggestion
FROM python:3-slim
```

---

If we go `python:3-slim` we'd need to add

```
RUN apt update && apt install -y gcc
```

If we go `python:3-alpine` we'd need to add

```
RUN apk add --no-cache gcc linux-headers musl-dev
```

Both of these require extra image deps in order to pre-install all of the other python deps. I think we should prioritize speed over image size since people will probably run these actions in their GitHub workflows which usually charge by time usage rather than resource usage iirc. What do ya think?

---

Ya sure - Stick with normal image then. Only suggested if it just worked to use it ... I avoid slim for the same reasons.